### ISSUE TITLE
Feature: configurable/hideable part fields

### ISSUE THREAD

**Issue by MightyPork** [rhaamo/StockazIO#370]
> make fields in the "new part" form configurable so it's not as long (possibly also hide them in other places). I only use like 4 of the fields.

**Comment by rhaamo** [comment]
I have researched customisable forms for a work thing and https://vueform.com/ could probably do the job, however that would still means a big bunch of custom stuff for some of our elements that needs custom handling (name, nearly all select, etc.)

so I'm not really interested in working on such a big change for now

**Comment by MightyPork** [comment]
yeah full custom would be difficult, i was thinking more like don't show some of the default stuff

**Comment by rhaamo** [comment]
I don't really know what to even remove from quick add
And hiding fields isn't either really possible without making everything even more complex
